tests/suites/test_suite_psa_crypto.function

```diff
@@ -294,6 +294,19 @@ exit:
     ((void) 0)
 #endif /* MBEDTLS_PSA_CRYPTO_SE_C */
 
+#define INPUT_INTEGER 0x10000   /* Out of range of psa_key_type_t */
+
+uint64_t parse_binary_string(data_t *bin_string)
+{
+    uint64_t result = 0;
+    TEST_LE_U(bin_string->len, 8);
+    for (size_t i = 0; i < bin_string->len; i++) {
+        result |= ((bin_string->x)[i]) << (i * 8);
```

Comment:
I missed this but the CI didn't:
```suggestion
        result |= ((uint64_t) (bin_string->x)[i]) << (i * 8);
```

But actually, no, that's not right either. I also missed that this is not the endianness I'd expect. If I see `"012345"` in the `.data` file, I expect it to mean 0x012345 (passed via a string because the argument can also be binary data rather than an integer, so we have to encode the integer for the sake of the test framework). But it actually means 0x452301. In other words, I expect a big-endian string of hex digits (and I think this is a natural expectation), but this function parses it as little-endian. Please change to big-endian.
```suggestion
        result = (uint64_t) result | bin_string->x[i];
```